srtcore/core.cpp

```diff
@@ -11653,6 +11886,10 @@ int CUDTGroup::sendBroadcast(const char* buf, int len, SRT_MSGCTRL& w_mc)
         m_iLastSchedSeqNo = curseq;
     }
 
+    // }
+
+    // { send_CheckBrokenSockets()
+
```

Comment:
Commented out code. Is it needed?

Reply:
Should be removed. These are markers I made for the sake of slashing this function into smaller pieces.